docs/zh/latest/CHANGELOG.md

```diff
@@ -40,6 +41,36 @@ title: CHANGELOG
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.5.0
+
+### Change
+
+- 更改 zipkin 插件的 span 类型 [#3877](https://github.com/apache/apisix/pull/3877)
+
+### Core
+
+- :sunrise: 支持 etcd 客户端证书校验 [#3905](https://github.com/apache/apisix/pull/3905)
+- :sunrise: 支持表达式使用“或”和“非”的逻辑 [#3809](https://github.com/apache/apisix/pull/3809)
+- :sunrise: 默认启动时会同步etcd配置 [#3799](https://github.com/apache/apisix/pull/3799)
+- :sunrise: 负载均衡支持节点优先级 [#3755](https://github.com/apache/apisix/pull/3755)
+- :sunrise: 服务发现提供了一系列 control API [#3742](https://github.com/apache/apisix/pull/3742)
+
+### Plugin
+
+- :sunrise: 允许热更新 skywalking 插件配置，并允许配置上报间隔 [#3925](https://github.com/apache/apisix/pull/3925)
+- :sunrise: consumer-restriction 支持 HTTP method 级别的白名单配置 [#3691](https://github.com/apache/apisix/pull/3691)
+- :sunrise: cors 插件支持通过正则表达式匹配 Origin [#3839](https://github.com/apache/apisix/pull/3839)
+- :sunrise: response-rewrite 插件支持条件改写 [#3577](https://github.com/apache/apisix/pull/3577)
+
+### Bugfix
```

Comment:
It seems that https://github.com/apache/apisix/pull/3948 was not enrolled?

Reply:
@tokers 
Only important changes are included. Otherwise, the list is too long.

Reply:
Got it.